lib/tools/pens/BoundsPen.js

```diff
@@ -0,0 +1,71 @@
+define(
+    [
+        'ufojs/tools/pens/ControlBoundsPen'
+      , 'ufojs/tools/misc/arrayTools'
+      , 'ufojs/tools/misc/bezierTools'
+    ],
+    function(
+        Parent
+      , arrayTools
+      , bezierTools
+) {
+    "use strict";
+    /**
```

Comment:
to make it better visible what our dependencies are:
```
    var updateBounds = arrayTools.updateBounds
      , pointInRect = arrayTools.pointInRect
      , unionRect = arrayTools.unionRect
      , calcCubicBounds = bezierTools.calcCubicBounds
     ;
```
and later use the names directly